raml-docs-method-viewer.html

```diff
@@ -230,6 +231,10 @@
       transform: rotateZ(-180deg);
     }
 
+    .title-area-actions {
+      @apply(--title-area-actions);
```

Comment:
Is that a theme thing or only for this element?
Because if this is a mixin that belongs to only this element it should be prefixed with element name. Otherwise we will be lost with variables very fast.